### PR TITLE
Added Betaflight Discord server link to the landing page

### PR DESCRIPTION
**Pull request by limonspb**
it's probably time :)

image

This server is created with the @mikeller  blessing some time ago. A part of the Betaflight core group is already there.
Drop me a message for admin/moderation rights after joining if you are interested: _limon (Ivan)#5712
Need moderators with Discord setting up experience.

Also changed the color of the language selector. Silver looked invisible on yellow.

### CODE REVIEW

File: locales/en/messages.json
Comment (haslinghuis) [inline locales/en/messages.json:716]:
```suggestion
    "defaultDiscordText": {
```

File: src/tabs/landing.html
Comment (haslinghuis) [inline src/tabs/landing.html:47]:
```suggestion
                    <div class="socialMediaText" i18n="defaultDiscordText"></div>
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: src/tabs/landing.html
Comment (limonspb) [reply src/tabs/landing.html]: lol

**Review by haslinghuis — APPROVED** [review]

**Comment by McGiverGim** [comment]
The decision is to move to discord?

If that is true, we must "lock" the old slack chat with a message to move to the new place. Having two places where the users can go to find support I think is not a good idea.

So my opinion is first move to the new place, later add the info in the Configurator.

**Comment by DusKing1** [comment]
Didn't we just vote on moving topic before? Most of us don't want move to discord.

**Comment by limonspb** [comment]
> The decision is to move to discord?
> 
> If that is true, we must "lock" the old slack chat with a message to move to the new place. Having two places where the users can go to find support I think is not a good idea.
> 
> So my opinion is first move to the new place, later add the info in the Configurator.

Not moving from slack. Just adding discord. And if it goes good, later we can discuss about removing slack if we need to.

And we already have multiple places where users can find help. Adding one more with infinite history, better popularity isn't anything bad, is it?

File: src/css/tabs/landing.css
Comment (chmelevskij) [inline src/css/tabs/landing.css:54]:
```suggestion
.text1, .text2, .text3 {
```

Comment (limonspb) [reply src/css/tabs/landing.css:54]: my bad =)

**Comment by haslinghuis** [comment]
Slack is for the community. Discord is developers only at the moment making zoom like calls possible.

**Comment by Zuldan** [comment]
Chat systems
 
Once Betaflight has migrated over to Discord, I can finally uninstall Slack.

**Comment by ctzsnooze** [comment]
I don't mind having multiple places to communicate, and some people prefer Discord, so I don't mind both.

I would value a private developer area on Discord for the video conferencing it provides.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
